Expensify/react-native-web
# Update repo with upstream changes

Just updating this repo from the upstream, and resolving conflicts.

Part of this fix: https://github.com/Expensify/App/issues/13968
Needed to fix this as well: https://github.com/Expensify/App/issues/13871

Preparation to test:
1. Build packages.
2. Drag and drop the packages on the App node-modules to replace the current ones.

Tests:
1. Open the app
2. Go to settings > About > View Keyboard shortcuts
3. Open the popup and close it
4. Again open the popup and try to select text
5. User should be able to select text
